tests/product_tests/foreman/foreman_tests.py

```diff
@@ -0,0 +1,100 @@
+# This file is part of the sos project: https://github.com/sosreport/sos
+#
+# This copyrighted material is made available to anyone wishing to use,
+# modify, copy, or redistribute it subject to the terms and conditions of
+# version 2 of the GNU General Public License.
+#
+# See the LICENSE file in the source distribution for further information.
+
+
+from sos_tests import StageOneReportTest, redhat_only
+
+# known values in our CI test images
+FOREMAN_DB_PASSWORD = r'S0Sdb=p@ssw0rd!'
+FOREMAN_ADMIN_PASSWORD = r'S0S@dmin\\p@ssw0rd!'
+KATELLO_DB_PASSWORD = r'S0SKatello%sp@ssw0rd!'
```

Comment:
Where is this used? Katello doesn't have a DB. In a Katello install there are 3 DBs: Foreman, Candlepin and Pulp (3). Each has their own password. Katello is a plugin for Foreman so it uses that DB.

Reply:
This was passed to `--katello-candlepin-db-password`, so I could rename it to `CANDLEPIN_DB_PASSWORD` if that tracks easier.

Reply:
It would. There's also (at least with pulp 3) `--foreman-proxy-content-pulp-db-password`. With Pulp 2 it's more complicated because in the katello/satellite scenario it's `--katello-something` and in the foreman-proxy-content/capsule it's `--foreman-proxy-content-something`.

At least in upstream Pulp 2 is only used in 3.18 and once Katello 4.1 goes GA that's EOL. Right now 4.1 rc1 is in the process of being released. That means that if you care about upstream and the future, the focus should be on Pulp 3.